VueJSApp/database/migrations/2018_07_26_105920_create_todo_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTodoTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('todo', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->string('note');
+            $table->mediumInteger('priority');
+            $table->boolean('checked');
```

Comment:
Vidi da sve boolean vrednosti pocinjes sa `is` ili `are` :) I checked je malo cudno, pre bi imalo smisla `is_done`